hack/shoot-comparator/pkg/shoot/matcher.go

```diff
@@ -140,61 +112,38 @@ func (m *Matcher) Match(actual interface{}) (success bool, err error) {
 			path:          "spec/secretBindingName",
 		},
 		{
-			GomegaMatcher: gomega.BeComparableTo(eShoot.Spec.SeedName),
-			expected:      aShoot.Spec.SeedName,
-			path:          "spec/seedName",
-		},
-		{
-			GomegaMatcher: gomega.BeComparableTo(eShoot.Spec.SeedSelector),
-			expected:      aShoot.Spec.SeedSelector,
-			path:          "spec/seedSelector",
-		},
-		{
-			GomegaMatcher: gomega.BeComparableTo(eShoot.Spec.Resources),
-			expected:      aShoot.Spec.Resources,
-			path:          "spec/resources",
-		},
-		{
-			GomegaMatcher: gomega.BeComparableTo(eShoot.Spec.Tolerations),
-			expected:      aShoot.Spec.Tolerations,
-			path:          "spec/tolerations",
-		},
-		{
-			GomegaMatcher: gomega.BeComparableTo(eShoot.Spec.ExposureClassName),
```

Comment:
`exposureClassName` is set by the [converter](https://github.com/kyma-project/infrastructure-manager/blob/main/internal/gardener/shoot/extender/exposure_classname_extender.go) so that we should compare it.